ci/redmine_bugzilla.py

```diff
@@ -149,12 +150,22 @@ def main():
                         links_issues_record += 'Redmine #%s -> Bugzilla %s, but Bugzilla %s does not ' \
                                            'link back\n' % (issue.id, bug.id, bug.id)
 
+    start_time = time.time()
+    redmine_issues_looked_up = 0
     for bug in bugzilla_bugs:
         for external_bug in bug.external_bugs:
                 if external_bug['type']['description'] == 'Pulp Redmine':
                     add_cc_list_to_bugzilla_bug(bug)
                     issue_id = external_bug['ext_bz_bug_id']
-                    issue = redmine.issue.get(issue_id)
+                    try:
+                        redmine_issues_looked_up += 1
+                        issue = redmine.issue.get(issue_id)
```

Comment:
You might consider writing a timing method which times and records timing information and data about the query.

The instead of `issue = redmine.issue.get(issue_id)` could become `issue = run_and_time(redmine.issue.get, [issue_id])`. More generally the signature would be `def run_and_time(callable, args, kwargs)`.